Makefile

```diff
@@ -64,6 +64,6 @@ test-integration:
 # Create a DB migration
 .PHONY: db-migration
 db-migration:
-	@if [[ "$(prefix)" == "" ]]; then echo "Invalid prefix, see example 'make db-migration prefix=26_v0.10.18'" && exit 1; fi;
+	@if [[ "$(prefix)" == "" ]]; then echo "Invalid prefix, see example 'make db-migration prefix=27_v0.10.20'" && exit 1; fi;
```

Comment:
Updated the script to use a prefix that we actually have (after merging this PR)

Reply:
There is no reason why I think we can't autogenerate the prefix, or at least propose a reasonable option.


Here is a bash one liner that grabs the latest release (for the repo the command is run in) then increments the patch version and stores it in the `nextVersion` variable
```bash
nextVersion=$(gh release view --json tagName --jq .tagName | awk -F. -v OFS=. '{$NF += 1 ; print}')
```

Reply:
Great suggestion! What do you think about using `git tag | sort -V | tail -1` instead of `gh release view`?
The latter requires having the GitHub CLI installed and authenticated. The former requires having recent tags fetched from the GitHub repository. The former might fail more gracefully as at least some tags should be available

Reply:
Extracted this into a separate PR https://github.com/cloudquery/cq-provider-aws/pull/794